src/functions-and-arrays.js

```diff
@@ -52,14 +163,44 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(wordsUnique) {
+if (!wordsUnique.length) {
+   return null;
+}else {
+  UniqueArr = [];
```

Comment:
As mentioned before always declare variable with const or let. Also use start your variable name with lowerCase. Not an error but a convention :)